pkg/simple/client/devops/jenkins/utils.go

```diff
@@ -91,23 +90,13 @@ func parseJenkinsQuery(query string) (url.Values, error) {
 		if i := strings.Index(key, "="); i >= 0 {
 			key, value = key[:i], key[i+1:]
 		}
-		key, err1 := url.QueryUnescape(key)
-		if err1 != nil {
-			if err == nil {
-				err = err1
+		if key, err = url.QueryUnescape(key); err == nil {
+			if value, err = url.QueryUnescape(value); err == nil {
+				result[key] = append(result[key], value)
```

Comment:
The previous processing will ignore the error, which causes an incorrect number?

Reply:
The problems come from https://github.com/kubesphere/kubesphere/pull/3335/files#diff-bdbc72911999a583181726d98c9931b3da7070145be9262a56c1b310f6cb9c5dR43